docs/user-guide/containers.md

```diff
@@ -94,3 +94,7 @@ The relationship between Docker's capabilities and [Linux capabilities](http://m
 | SETFCAP |  CAP_SETFCAP |
 | WAKE_ALARM |  CAP_WAKE_ALARM |
 | BLOCK_SUSPEND |  CAP_BLOCK_SUSPEND |
+
+Here is an example pod that uses capabilities to allow a container to change its hostname:
```

Comment:
@devin-donnelly, No problem, as capabilities are a concept from the Linux Kernel, I'll update to specify Docker capabilities. Though these concepts and syntax are also applicable to other Container runtimes. eg RKT.